Marlin/Configuration.h

```diff
@@ -1684,6 +1684,18 @@
 // If the servo can't reach the requested position, increase it.
 #define SERVO_DELAY { 300 }
 
+/**
+ * Default extrusion settings
+ *
+ * These settings control basic extrusion from within the Marlin firmware.
+ * 
+ */
+#define DEFAULT_NOMINAL_FILAMENT_DIA 3.00   // (mm) Diameter of the filament generally used (3.0 or 1.75mm), also used in the slicer. Used to validate sensor reading.
+#define DEFAULT_NOZZLE_SIZE           .4    // (mm) Diameter of primary nozzle.  Used by G26 Mesh Validation Pattern tool.
+#define DEFAULT_LAYER_HEIGHT          .2    // (mm) Default layer height that will produce usable results by the printer.  Used by G26 Mesh Validation Pattern tool.
+#define DEFAULT_HOTEND_TEMP        205.0    // (c)  Default nozzle temperature that will produce usable results by the printer.  Used by G26 Mesh Validation Pattern tool.
+#define DEFAULT_BED_TEMP            60.0    // (c)  Default bed temperature that will produce usable results by the printer.  Used by G26 Mesh Validation Pattern tool.
+
```

Comment:
Perhaps since these options only apply to the single function `G26` they shouldn't be implied to be globally important, and should be sub-options of the G26 configuration option. The `DEFAULT_NOMINAL_FILAMENT_DIA` is specifically about volumetric extrusion calculation in relation to a filament width sensor, and is not a general option about filament diameter. That's why it has `NOMINAL` in the name.

Is there a corresponding 1.1 patch with these changes? I didn't see it linked from the 2.0.x PR.